src/main/java/org/folio/consortia/config/AppConfig.java

```diff
@@ -37,7 +36,6 @@ public TaskExecutor asyncTaskExecutor() {
     executor.setMaxPoolSize(Runtime.getRuntime().availableProcessors() * 2);
     executor.setQueueCapacity(500);
     executor.setThreadNamePrefix("ConsortiaAsync-");
-    executor.setTaskDecorator(FolioExecutionScopeExecutionContextManager::getRunnableWithCurrentFolioContext);
```

Comment:
Please update all existing usages of task executor to pass folioExecutionContext to them

Reply:
Just for information for reviewers: we deleted this decorator, because getRunnableWithCurrentFolioContext was setting some member tenant context, and we need to set central tenant context before starting asynchronus code